en/examples/follow_me.md

```diff
@@ -0,0 +1,341 @@
+# Example: Follow Me Mode
+
+This example demonstrates how to use the [Follow Me](../api_reference/classdronecore_1_1_follow_me.md) plugin. 
+It shows how to send the drone both the current position of the target (`FollowMe::TargetLocation`) and the relative position at which it should follow (`FollowMe::Config`).
+
+![Follow Me QGC Screenshot](../../assets/examples/follow_me/follow_me_example_qgc.jpg)
```

Comment:
@shakthi-prashanth-m I need a screenshot here.

Reply:
![followme](https://user-images.githubusercontent.com/26615772/36210759-492c1c3a-11c5-11e8-8011-96ae8e1406b9.png)

Reply:
@hamishwillee I built QGC source code by disabling it sending Follow positions. This made PX4 to receive Follow position only from DroneCore Follow Me example. But Follow Me trajectory is not in a regular shape (I expected this because as I mentioned it as a rudimentary location provider :-) ). So is this fine ?